std/range/package.d

```diff
@@ -6590,11 +6595,53 @@ if (isForwardRange!RangeOfRanges &&
     }
 
     auto opSlice() { return this; }
+    
+    static if (hasSlicing!(ElementType!RangeOfRanges))
```

Comment:
Right. No slicing, then. Could be done by saving offsets in the struct and having the cost per element lookup, but that complicates the code for very little benefit.